File: remind-me-query/src/main/java/com/dutra/remind/me/query/RemindMeQueryApplication.java
package com.dutra.remind.me.query;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cloud.netflix.eureka.EnableEurekaClient;

@SpringBootApplication
@EnableEurekaClient
public class RemindMeQueryApplication {

	public static void main(String[] args) {
		SpringApplication.run(RemindMeQueryApplication.class, args);
	}
}

File: remind-me-query-kotlin/src/main/kotlin/com/dutra/remind/me/query/RemindMeQueryApplication.kt
package com.dutra.remind.me.query

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.cloud.netflix.eureka.EnableEurekaClient

@SpringBootApplication
@EnableEurekaClient
class RemindMeQueryApplication

fun main() {
    runApplication<RemindMeQueryApplication>()
}